.github/workflows/cdr-infra.yml

```diff
@@ -20,13 +20,35 @@ on:
         description: "The version of the ELK stack: For BC use version without hash 8.x.y, for SNAPSHOT use 8.x.y-SNAPSHOT"
         default: "8.16.0"
         type: string
+      cis-infra:
+        required: false
+        description: "Deploy the CIS infrastructure"
+        type: boolean
+        default: false
```

Comment:
why do we need CIS infra inside the CDR workflow? aren't they supposed to be separated?

Reply:
After refactoring the Create Environment workflow, we can provide a choice, such as a combo box with options to select CIS, CDR, or All. From the CDR perspective, we only need to specify whether we want to install CIS as well. In this case, I believe a checkbox would be preferable.